Source/ProgressionSystemRuntime/Public/Data/PSTypes.h

```diff
@@ -69,3 +61,22 @@ struct FPSRowData : public FTableRowBase
 	UPROPERTY(EditAnywhere, BlueprintReadWrite, Category="C++")
 	TObjectPtr<class UCurveTable> MenuStarsAnimation = nullptr;
 };
+
+USTRUCT(BlueprintType)
```

Comment:
above every struct, class, function and property should be your explanation how does it work. In case of SaveToDiskData, please explain approach for others, like say it contains values that supposed to be changed in runtime and save to disk with every modification

Reply:
fixed the old struct description and added for new: https://github.com/h4rdmol/ProgressionSystem/pull/18/commits/c34e1d39b3890283bee65b818e926bbbcae01ab6